src/org/hammerhead226/sharkmacro/Parser.java

```diff
@@ -148,12 +148,29 @@ protected boolean writeToFile(ArrayList<String[]> data) {
 		}
 
 		List<String[]> rawFile = parseCSV(filename);
-		
+
 		cache.put(filename, rawFile);
-		
+
 		return rawFile;
 	}
 
+	/**
+	 * Add all CSV files in a directory to the cache.
+	 * 
+	 * @param directory
+	 *            the directory the files are located in
+	 */
+	protected static void cacheAll(String directory) {
+		File[] files = new File(directory).listFiles();
+
+		for (File f : files) {
+			if (f.isFile()) {
+				cache(directory, f.getName());
+			}
+		}
+
+	}
```

Comment:
Not as familiar with the Java libs, probably want to make sure `directory` actually exists first, or consider some try/catch.